core/controlplane/config/encrypted_assets.go

```diff
@@ -440,39 +440,46 @@ func ReadOrEncryptAssets(dirname string, manageCertificates bool, caKeyRequiredO
 }
 
 func (r *RawAssetsOnMemory) WriteToDir(dirname string, includeCAKey bool) error {
+	workerCAKeyDefaultSymlinkTo := ""
+	if includeCAKey {
+		workerCAKeyDefaultSymlinkTo = "ca-key.pem"
+	}
+
 	assets := []struct {
-		name      string
-		data      []byte
-		overwrite bool
+		name             string
+		data             []byte
+		overwrite        bool
+		ifEmptySymlinkTo string
```

Comment:
Does this mean if populated, create a symlink?

Reply:
If we had nothing to write, default to create a symlink to denoted by the `ifEmptySymlinkTo`.
Basically, we have nothing to write for `worker-ca.pem` as long as the user explicitly overwrite or put the file before running `kube-aws render credentials`. So in that case, create a symlink to `ca.pem`(=the same as the previous behavior).